### ISSUE TITLE
react-infinite-scroll-component is no longer maintained

### ISSUE THREAD

**Issue by IhnatKlimchuk** [ankeetmaini/react-infinite-scroll-component#388]
I see that [repo](https://github.com/ankeetmaini/react-infinite-scroll-component) is not longer maintained while suffering from major issues and not adding more functionality like "error component, in case next failed". 

@ankeetmaini can you please notify if you will to maintain it?

In case no reply comes in 1 month, I would suggest to create a new main fork with new npm package and maintainers willing to cooperate.

**Comment by CharlyJazz** [comment]
Sounds logic

**Comment by 514sid** [comment]
I agree with the concerns you've raised about this repository, @IhnatKlimchuk.

Unfortunately, despite waiting for a month and reaching out to @ankeetmaini for a response regarding maintaining the repository, we haven't received any communication.

I think it's time to create a fork.

**Comment by 514sid** [comment]
For anyone looking for a React infinite scroll implementation, I suggest using React Query with Intersection Observer.

I tried all the popular React infinite scroll packages, and all of them were either not well-maintained or had some bugs. This solution is much better and simple enough to use.

Here is an example: https://tanstack.com/query/v4/docs/react/examples/react/load-more-infinite-scroll

**Comment by nikeee** [comment]
Any news on the project status or an established fork?

**Comment by 514sid** [comment]
@zeraye thanks!

**Comment by iamdarshshah** [comment]
The library is actively maintained. I joined @ankeetmaini as a co-maintainer, and v7.1.0 is shipping with a full rewrite: function component, `IntersectionObserver`-based detection, dropped `throttle-debounce` dependency, React 18/19 support, SSR guard for Next.js App Router, and fixes for the most-reported bugs. Closing this as the concern is addressed. Feel free to reopen if you have specific maintenance concerns going forward.
